pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -69,6 +69,8 @@ core-packages:
 default-package-overrides:
   # pandoc-2.9 does not accept the 0.3 version yet
   - doclayout < 0.3
+  # gdk 4 is not provided
+  - gi-gdkx11 < 4
```

Comment:
What does this mean that gdk 4 is not provided?

Can we instead provide gdk4 to `gi-gdkx11`?

Reply:
Sorry, it comment is a bit misleading, (and there is a typo it should be `gtk4`) better would be gtk4 is still in development (see https://github.com/haskell-gi/haskell-gi/issues/269#issuecomment-569453210)

I'll correct that and update the pull request